src/sentry/api/endpoints/event_attachment_details.py

```diff
@@ -57,22 +56,17 @@ class EventAttachmentDetailsEndpoint(ProjectEndpoint):
     permission_classes = (EventAttachmentDetailsPermission,)
 
     def download(self, attachment):
-        file = File.objects.get(id=attachment.file_id)
-
-        content_type = attachment.content_type or file.headers.get(
-            "content-type", "application/octet-stream"
-        )
-        size = attachment.size or file.size
+        fp = attachment.getfile()
+        name = posixpath.basename(" ".join(attachment.name.split()))
 
-        fp = file.getfile()
-        response = StreamingHttpResponse(
-            iter(lambda: fp.read(4096), b""),
-            content_type=content_type,
+        response = FileResponse(
+            fp,
+            as_attachment=True,
```

Comment:
Is it a problem that `Content-Disposition` will now be `"attachment"` instead of `"attachment; filename=foo.txt"`?

Reply:
This still works as expected, and its verified by the tests